app/uploaders/lighthouse_document_uploader_base.rb

```diff
@@ -0,0 +1,45 @@
+# frozen_string_literal: true
+
+class LighthouseDocumentUploaderBase < CarrierWave::Uploader::Base
```

Comment:
EVSS used an uploader, but I may be having a misunderstanding of how the Documents API handles uploads. We may be able to skip the whole uploader altogether